apps/lead/tests/test_apis.py

```diff
@@ -81,6 +81,38 @@ def test_create_lead(self, assignee=None):
         self.assertEqual(r_data['title'], data['title'])
         self.assertEqual(r_data['assignee'], self.user.id)
 
+        # low is default priority
+        self.assertEqual(r_data['priority'], Lead.LOW)
+
+    def test_create_high_priority_lead(self, assignee=None):
+        lead_count = Lead.objects.count()
+        project = self.create(Project, role=self.admin_role)
+
+        url = '/api/v1/leads/'
+        data = {
+            'title': 'Spaceship spotted in sky',
+            'project': project.id,
+            'source': self.source.pk,
+            'author': self.author.pk,
+            'confidentiality': Lead.UNPROTECTED,
+            'status': Lead.PENDING,
+            'text': 'Alien shapeship has been spotted in the sky',
+            'assignee': assignee or self.user.id,
+            'priority': Lead.HIGH,
+        }
+
+        self.authenticate()
+        response = self.client.post(url, data)
+        self.assert_201(response)
+
+        self.assertEqual(Lead.objects.count(), lead_count + 1)
+        r_data = response.json()
+        self.assertEqual(r_data['title'], data['title'])
+        self.assertEqual(r_data['assignee'], self.user.id)
+
+        # low is default priority
+        self.assertEqual(r_data['priority'], Lead.HIGH)
+
```

Comment:
@frozenhelium Please add separate tests for sorting and filter too. Thanks

Reply:
done